prow/cmd/exporter/collector.go

```diff
@@ -91,6 +94,23 @@ func (pjc prowJobCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 }
 
+func getLatest(jobs []*prowapi.ProwJob) map[string]*prowapi.ProwJob {
+	latest := map[string]time.Time{}
+	latestJobs := map[string]*prowapi.ProwJob{}
+	for _, job := range jobs {
+		if _, ok := latest[job.Spec.Job]; !ok {
```

Comment:
When `!ok` you get  a zero-value time, and no matter what `.After(time.Time{})` will be true, so you don't need this `if` block

https://play.golang.org/p/PuKeKf1Rk3n

Reply:
That is to deal with the case (maybe not a valid case at all ^_^).

https://github.com/kubernetes/test-infra/blob/b69861b8202cba9601955934eff91c5e7174dc47/prow/cmd/exporter/collector_test.go#L145-L149